modules/videoio/src/cap_ffmpeg_legacy_api.hpp

```diff
@@ -30,7 +30,7 @@ OPENCV_FFMPEG_API int cvSetCaptureProperty_FFMPEG(struct CvCapture_FFMPEG* cap,
 OPENCV_FFMPEG_API double cvGetCaptureProperty_FFMPEG(struct CvCapture_FFMPEG* cap, int prop);
 OPENCV_FFMPEG_API int cvGrabFrame_FFMPEG(struct CvCapture_FFMPEG* cap);
 OPENCV_FFMPEG_API int cvRetrieveFrame_FFMPEG(struct CvCapture_FFMPEG* capture, unsigned char** data,
-                                             int* step, int* width, int* height, int* cn);
+                                             int* step, int* width, int* height, int* cn, const int flag);
```

Comment:
We should not change already existed API.
Legacy API should be used with `flag=0` only.

`flag != 0` should be handled differently.
Please take a look on addition of `retrieveHWFrame` implementation (which has missing `flag` parameter too...)

Reply:
👍